includes/fs-plugin-info-dialog.php

```diff
@@ -328,7 +328,15 @@ function _get_addon_info_filter( $data, $action = '', $args = null ) {
                 }
             }
 
-            $data->sections['description'] = fs_get_template( '/plugin-info/description.php', $view_vars );
+            $description = fs_get_template( '/plugin-info/description.php', $view_vars );
+            $description = trim( $description );
+
+            $description = $this->_fs->apply_filters( 'plugin_section_description', $description, $data );
+
+            // Only override description if it's not empty or not set yet.
+            if ( '' !== $description || ! isset( $data->sections['description'] ) ) {
+                $data->sections['description'] = $description;

```

Comment:
This logic will allow the WordPress.org plugin description to not be replaced with an empty Freemius add-on description.